File: app/src/main/java/Model/Category.java
package Model;

import java.util.List;

public class Category {

    private String title;
    private int resDrawable;
    private List<Product> itemList;

    public Category() {
    }

    public Category(String title, int resDrawable) {
        this.title = title;
        this.resDrawable = resDrawable;
    }

    public List<Product> getItemList() {
        return itemList;
    }

    public void setItemList(List<Product> itemList) {
        this.itemList = itemList;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setResDrawable(int resDrawable) {
        this.resDrawable = resDrawable;
    }

    public int getResDrawable() {
        return resDrawable;
    }

    public String getTitle() {
        return title;
    }

}

File: app/src/main/java/Model/Category.kt
package Model

class Category {
    private var title: String? = null
    private var resDrawable = 0
    private var itemList: MutableList<Product?>? = null

    constructor()
    constructor(title: String?, resDrawable: Int) {
        this.title = title
        this.resDrawable = resDrawable
    }

    fun getItemList(): MutableList<Product?>? {
        return itemList
    }

    fun setItemList(itemList: MutableList<Product?>?) {
        this.itemList = itemList
    }

    fun setTitle(title: String?) {
        this.title = title
    }

    fun setResDrawable(resDrawable: Int) {
        this.resDrawable = resDrawable
    }

    fun getResDrawable(): Int {
        return resDrawable
    }

    fun getTitle(): String? {
        return title
    }
}